### PR TITLE
Javascript/javascript3/week2

### PR DESCRIPTION
**Pull request by hamedveloper**
Homework JS3 week2. I am waiting for your review dear mentor:)

### CODE REVIEW

**Comment by bmja62** [comment]
Hi Hamed, 
Thank you for your great job. 
I can see here there are your files for two homework and I am not sure if it is the right way to do. 
so could you please create two PR and then remove none related files.

I will review your code when you have removed files and have created a new pull request.

Best regards,
Bashir

**Comment by hamedveloper** [comment]
Thank you Bashir, 

I have already make them correct. Whould you please check them and let me know.

File: javascript/javascript3/week2/JS3W2.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <script src="JS3W2.js"></script>
```
Comment (bmja62) [inline javascript/javascript3/week2/JS3W2.html:11]: it is much better if you keep the file name in lower case.

File: javascript/javascript3/week2/JS3W2.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+setTimeout(giveWeather, 3000);
+setTimeout(giveWeather1, 3000);
+
+function giveWeather() {
+    fetch('https://api.openweathermap.org/data/2.5/weather?q=tehran&appid=b33af74225294d0f38ecf99fd7eaca9a')
+        .then(answer => answer.json())
+        .then(weather => console.log(weather))
+}
+
+async function giveWeather1() {
```
Comment (bmja62) [inline javascript/javascript3/week2/JS3W2.js:79]: the difference is not much clear with giveWeather. a better name could improve the readability.

File: javascript/javascript3/week2/Visual Promise/main.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            x: 400 - parseInt(blue.style.left),
+            y: 300 - parseInt(blue.style.top)
+        }).then(() => moveElement(green, {
+            x: 400 - parseInt(green.style.left),
+            y: 20 - parseInt(green.style.top)
+        })).then(() => console.log('Elements has been moved'))
+    })
+}
+translateAllAtOnce()
+//translateOneByOne()
```
Comment (bmja62) [inline javascript/javascript3/week2/Visual Promise/main.js:38]: removing commented code is good practice.

**Review by bmja62 — APPROVED** [review]
Great job, I just made a few comments just for improving the code. Well done

**Pull request merged**